Makefile

```diff
@@ -76,6 +76,10 @@ docker:
 docker-push:
 	docker push $(HUB)/$(PROJECT):$(VERSION)
 
+.PHONY: docker-release
+docker-release: docker docker-push
```

Comment:
We may need to consider (seriously) providing multi-architecture Docker images 🤔